source/extensions/quic_listeners/quiche/platform/quic_socket_address_impl.h

```diff
@@ -10,7 +10,7 @@
 
```

Comment:
Since QuicSocketAddress has been de-platformized, can we delete this file now?

Reply:
QuicSocketAddress has not been de-platformized in this tar ball yet. I still has problem to roll the tar ball upto where it is de-platformized, waiting for upstream fix to be merged into quiche. But quic_ip_address_impl.h should.